projects/perception/face_recognition/demos/benchmarking_demo.py

```diff
@@ -25,40 +25,45 @@
     parser.add_argument("--onnx", help="Use ONNX", default=False, action="store_true")
     parser.add_argument("--backbone", help="Backbone to use (mobilefacenet, ir_50)", type=str, default='mobilefacenet')
     parser.add_argument("--device", help="Device to use (cpu, cuda)", type=str, default="cuda")
+    parser.add_argument("--create_new", help="Whether to create or load a database", type=bool, default=True)
     args = parser.parse_args()
 
     onnx, device, backbone = args.onnx, args.device, args.backbone
-
+    nvml = False
+    try:
+        if 'cuda' in device:
+            from pynvml import nvmlInit, nvmlDeviceGetMemoryInfo, nvmlDeviceGetHandleByIndex
+            nvmlInit()
+            nvml = True
+    except ImportError:
+        pass
```

Comment:
Here it should print an error message instead

Reply:
Pynvml is not being installed by default, maybe a message like 'You can install pynvml to also monitor the allocated gpu memory' sounds ok?